packages/editor-ui/src/router.ts

```diff
@@ -107,6 +107,9 @@ export const routes = [
 			middleware: ['authenticated'],
 		},
 	},
+	// Single workflow view, this is no longer used in-app
```

Comment:
that's not true. it's still used if you have an env variable. otherwise, we should delete it